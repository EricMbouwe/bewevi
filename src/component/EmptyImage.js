import emptyImageSVG from '../assets/emptyImage.svg';

const EmptyImage = ({ className = '' }) => (
  <img src={emptyImageSVG} alt="Not provided" className={className} />
);
export default EmptyImage;
