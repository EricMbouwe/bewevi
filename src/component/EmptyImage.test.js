import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import EmptyImage from './EmptyImage';

describe('<EmptyImage />', () => {
  it('renders an image', () => {
    const renderedComponent = renderer.create(
      <EmptyImage className="empty-image" />,
    ).toJSON();
    expect(renderedComponent.props.className).toEqual('empty-image');
  });

  it('renders correctly', () => {
    const tree = renderer.create(
      <EmptyImage
        className="empty-image"
      />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
