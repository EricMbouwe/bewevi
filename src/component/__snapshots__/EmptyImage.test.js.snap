// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyImage /> renders correctly 1`] = `
<img
  alt="Not provided"
  className="empty-image"
  src="emptyImage.svg"
/>
`;
